src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -22,25 +22,41 @@ export const defaultMinigraphOptions: MinigraphOptions = {
 
 let stepY = 5;
 
-export const DataflowNodePlot = (props: NodePlotProps) => {
+export const DataflowNodePlot: React.FC<INodePlotProps> = (props) => {
+  const [scalar, setScalar] = useState(1);
   if (!props.display) return null;
 
+  const handleClickScalar = (newScalar: number) => {
+    setScalar((oldVal) => oldVal * newScalar);
+  };
+  const scaleBtnColorClass= props.data.name.charAt(0).toLowerCase() + props.data.name.slice(1);
```

Comment:
```suggestion
  const scaleBtnColorClass = props.data.name.toLowerCase();
```
This should make the entire string lowercase. You don't want to preserve uppercase characters after the first, right?

Reply:
Since the `props.data.name` comes in as the node name capitalized  ("Sensor", "Number", "Generator" etc) - I'm making the first letter lowercase.
So  to answer your first point - yes this should make the entire string lowercase.
And for the question-  in theory there shouldn't ever be uppercase characters after the first.